src/types/Person.ts

```diff
@@ -3,8 +3,8 @@ export interface Person {
   sex: string,
   born: number,
   died: number,
-  fatherName: string | null,
-  motherName: string | null,
+  fatherName: string,
+  motherName: string,
```

Comment:
motherName should be nullable with 'string | null' type, as it was in the original code.